public/index.js

```diff
@@ -282,11 +286,18 @@ recipeButtons.forEach((button) => {
 
             const utterance = new SpeechSynthesisUtterance();
 
+            const speechBtns = Array.from(
+              document.querySelectorAll(".fa-solid")
+            );
+            const speedBtn = document.querySelector("#speed");
+
+            console.log(speechBtns);
+
             function readRecipe(recipe) {
-              if(speechSynthesis.paused && speechSynthesis.speaking) {
+              if (speechSynthesis.paused && speechSynthesis.speaking) {
```

Comment:
This is interesting. I've never encountered ".paused" or ".speaking" in Javascript. I didn't realise they existed! @josueJURE

Reply:
have a look at this link

https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesis